SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -0,0 +1,522 @@
+Source and Target cloner image for host assisted cloning
+
+%package        controller
+Summary:        Controller for the data fetching service
+Group:          System/Packages
+
+%description    controller
+Controller for the data fetching service for VM container images
+
+%package        importer
+Summary:        Data fetching service
+Group:          System/Packages
+
+%description    importer
+Data fetching service for VM container imagess
+
+%package        operator
+Summary:        Operator for the data fetching service
+Group:          System/Packages
+
+%description    operator
+Operator for the data fetching service for VM container images
+
+%package        uploadproxy
+Summary:        Upload proxy for the data fetching service
+Group:          System/Packages
+
+%description    uploadproxy
+Upload proxy for the data fetching service for VM container images
+
+%package        uploadserver
+Summary:        Upload server for the data fetching service
+Group:          System/Packages
+
+%description    uploadserver
+Upload server for the data fetching service for VM container images
+
+%package        manifests
+Summary:        YAML manifests used to install CDI
+Group:          System/Packages
+
+%description    manifests
+This contains the built YAML manifests used to install CDI into a
+kubernetes installation with kubectl apply.
+
+%package -n     obs-service-cdi_containers_meta
+Summary:        CDI containers meta information (build service)
+Group:          System/Packages
+
+%description -n obs-service-cdi_containers_meta
+The package provides meta information that is used during the build of
+the CDI container images.
+
+%prep
+# Unpack the sources respecting the GOPATH directory structure expected by the
+# go imports resolver. I.e. if DIR is in GOPATH then DIR/src/foo/bar can be
+# imported as "foo/bar". The same 'visibility' rules apply to the local copies
+# of external dependencies placed in 'vendor' directory when imported from the
+# 'parent' package.
+#
+# Note: having bar symlink'ed to DIR/src/foo/bar does not seem to work. Looks
+# like symlinks in go path are not resolved correctly. Hence the sources need
+# to be 'physically' placed into the proper location.
+%setup -q -n go/src/kubevirt.io/%{name} -c -T
+tar --strip-components=1 -xf %{SOURCE0}
+
+%build
+# Hackery to determine which registry path to use in cdi-operator.yaml
+# when building the manifests
+#
+# The 'kubevirt_registry_path' macro can be used to define an explicit path in
+# the project config, e.g.
+#
+# Macros:
+# %kubevirt_registry_path registry.opensuse.org/Virtualization/container
+# :Macros
+#
+# 'kubevirt_registry_path' can also be defined when building locally, e.g.
+#
+# osc build --define='kubevirt_registry_path registry.opensuse.org/foo/bar/baz' ...
+#
+# If 'kubevirt_registry_path' is not specified, the standard publish location for
+# SLE and openSUSE-based containers is used.
+#
+distro='%{?sle_version}:%{?is_opensuse}%{!?is_opensuse:0}'
```

Comment:
Please remove all the opensuse related statements. We don't require them.

Reply:
OpenSUSE specific scripts have been removed.